modules/commentCtr.js

```diff
@@ -0,0 +1,6 @@
+const comments = async (count) => {
+  const h4 = document.getElementById('comments');
+  h4.innerHTML = `Comments(${count})`;
+};
```

Comment:
- [ ] Great job on the UI and all the functionalities. :+1: Let's work on the counter functions to achieve the project requirement. Currently, this function is getting some number and displaying it on the screen. I think the counting is done in the other file `getComments.js`.
 
![Screenshot from 2023-03-22 15-13-24](https://user-images.githubusercontent.com/17243325/226903251-d17588ad-7039-470a-9a47-70d714b34f8a.png)

I believe the counting should be done inside the counter function and displaying the count should be in the other file, that way it'll make it easier to test the counters.

**Possible solution**

`commentCtr.js`

![Screenshot from 2023-03-22 15-27-19](https://user-images.githubusercontent.com/17243325/226905270-f821dd20-8f44-4a4f-880d-06b5d952a381.png)

`getComments.js`

![Screenshot from 2023-03-22 15-27-39](https://user-images.githubusercontent.com/17243325/226905390-a2d5d0d4-2acd-4b74-a966-44eb7d681e5f.png)